sdks/go/OWNERS

```diff
@@ -1,4 +1,5 @@
 # See the OWNERS docs at https://s.apache.org/beam-owners
 
 reviewers:
-  - herohde
+  - herohde # emeritus
```

Comment:
I think it only makes sense to keep the entry here, if Henning still wants to review. But then I don't see a reason to declare him emeritus :)

Reply:
True, he hasn't noticed many (which means I've done a good job catching the couple I saw), but he's happy being marked emeritus.

The trick is I'd like him to say so on this github PR, rather than in a private chat. :D

Reply:
LGTM. Fine to remove me as well, if that is preferable. I don't check GitHub regularly.

Reply:
There he is :) I'm also ok with keeping past reviewers. I think it would make sense to eventually have a separate emeritus section. At some point we also need to evaluate how well these OWNERS files work.